### ISSUE TITLE
moving tabs causes them to disappear 

### ISSUE THREAD

**Issue by vohzi** [piroor/treestyletab#1926]
HI,

When I move a tree of tabs with drag & drop in FF60 (windows 7 64 bits) it sometimes completely disappears from the left sidebar. I checked and rechecked, it's not hidden in another tree or something, it's just gone.

**Comment by piroor** [comment]
I couldn't reproduce it. Could you describe more detailed steps to reproduce based on the ISSUE_TEMPLATE.md? (please don't ignore that)

**Comment by Sada-Mio** [comment]
I experience this bug often when moving tabs from one window to another. All the tabs go hidden, but once I create a new tab it repopulates.

**Comment by piroor** [comment]
@Sada-Mio In your case, is the problem caused by the "Move Tab to New Window" command or, dragging a tree to another existing window? Tabs in Firefox's tab bar actually moved, or stay in original window?

**Comment by Sada-Mio** [comment]
@piroor I drag and drop to another existing window. Tabs actually move, but sometime need to refresh TST sidebar for it to update tree.

**Comment by Sada-Mio** [comment]
@vohzi do you happen to have [panorama view](https://addons.mozilla.org/en-US/firefox/addon/panorama-view/) addon installed?

I installed FF Nightly 62 with TST https://piro.sakura.ne.jp/xul/xpi/nightly/treestyletab-we.xpi and tried to drag a tree from one window to another. The tabs moved over, but all tabs went missing/hidden. A pop up alerted me that Panorama view addon has hidden my tabs. Once I create a new tab the hidden tabs come back.

**Comment by vohzi** [comment]
Hello and sorry for the late post. 

Some days past, I restarted firefox a bunch of times (like 5), don't know what happened, but the tabs are back where they should be now !
I have many tabs open usually, it runs like a charm (even though I can't have 2 sidebars at the same time anymore, 1 for my tabs, 1 for my bookmarks but I digress) 

I'm sorry @piroor I found those 2 steps below confusing and didn't want to mess my existing FF installation up
 1. Start Firefox with clean profile.
 2. Install TST.

@Sada-Mio 
I have a few other add-ons, but no panorama of any sorts as I recall, is there an easy way I can list them ?
